### ISSUE TITLE
streaming 

### ISSUE THREAD

**Issue by enrique0221** [drejkim/edi-cam#18]
hello,
first of all i want to thank you for this tutorial. i also want to ask you if you can explain how to from the cloning part an on. i been trying to stream but i can not get pass the installing the ffmpeg.

**Comment by estherjk** [comment]
Which parts are you specifically having trouble with? For example, are you experiencing issues installing ffmpeg?

**Comment by enrique0221** [comment]
yes. from that part an on i am not able to fallow the code. i have the camera install and i have done the test you recommend and they all return the same way as yours. however, after that i am lost

**Comment by enrique0221** [comment]
for example, what do you mean by "navigate to bin"

**Comment by estherjk** [comment]
Ah, what I mean by "navigate to bin" is to change to the `bin` directory. For example, if you are already in the `edi-cam` directory, you would type `cd bin`. Hope this clarifies!

**Comment by enrique0221** [comment]
okay. i guess my problem now is when i get there. what am i supposed to do?

**Comment by enrique0221** [comment]
im sorry, i am so confuse. for this part of the code am i still on the edison? am i supposed to install the ffmpeg on the edison?

**Comment by estherjk** [comment]
Yes, all of the steps refer to installing on Edison!

**Comment by taimurIslam** [comment]
hello,
can u explain why my browser not showing the video stream?? 
i find this output in my console, 
root@Group14:~/edi-cam/web/server# node server.js
WebSocket server listening on port 8084
HTTP server listening on port 8080
Listening for video stream on port 8082
Stream Connected: ::ffff:127.0.0.1:42243 size: 320x240
-----------------------
but when i hit http://group14.local:8080 there is no video stream :( :(

**Comment by MunKit** [comment]
@taimurIslam what did it show on the console? did it show any error?
